ppr-ui/tests/unit/UnitNotePanels.spec.ts

```diff
@@ -37,6 +38,88 @@ function createComponent (): Wrapper<any> {
   })
 }
 
+const verifyHeaderContent = (note: UnitNoteIF, header: Wrapper<any>) => {
+  // Check the unit note type
+  const typeText = header.find('h3').text()
+
+  let statusText = ''
+  if (note.status === UnitNoteStatusTypes.CANCELLED) {
+    statusText = ' - Cancelled'
+  } else if (note.status === UnitNoteStatusTypes.EXPIRED) {
+    statusText = ' - Expired'
+  }
+
+  const expectedTypeText = (UnitNotesInfo[note.documentType]?.panelHeader ??
+  UnitNotesInfo[note.documentType].header) + statusText
+  expect(typeText).toBe(expectedTypeText)
+
+  // Check the registration number and date
+  const registrationInfo = header.find('.info-text')
+
+  expect(registrationInfo.exists()).toBe(true)
+  expect(registrationInfo.text()).toContain(`Registered on ${pacificDate(note.createDateTime, true)}`)
+  expect(registrationInfo.text()).toContain(`Document Registration Number ${note.documentRegistrationNumber}`)
+}
+
+const verifyBodyContent = (note: UnitNoteIF, content: Wrapper<any>) => {
```

Comment:
I'm not a big fan of testing the actual content, since it can change frequently. But ok with it, since the work is already done.

Reply:
Sounds good will note for the future.